modules/ROOT/partials/commands/footnotes-cmds.adoc

```diff
@@ -0,0 +1,13 @@
+[cols="1,3",options="header"]
+|===
+|Command |Description
+|mceInsertFootnote |Inserts a superscripted footnote number into the editor.
```

Comment:
This description is different to what's used in the menu item/toolbar, even though they do the same thing.